ui/components/schema/SchemaContainer.js

```diff
@@ -105,17 +77,17 @@ class SchemaContainer extends Component {
         return (
             <React.Fragment>
                 <CommonToolbar buttons={this.getToolbarButtons()} />
-                <div ref={this.setflexWrapperRef} className={style.flexWrapper} style={{ height: this.state.height }}>
-                    <div className={style.left}>
+                <Row>
+                    <Col xs={8} style={{ paddingRight: "0px" }}>
```

Comment:
Again I am just wondering about referencing css, is this way just because one small piece of css is all that is needed?